github_client/github.go

```diff
@@ -1,11 +1,12 @@
-package github
+package github_client
```

Comment:
[Line 101-103] Potential Bug: The indirect dependencies are missing and others are added with specific versions. Please verify that these specific versions do not cause any compatibility issues with other parts of the codebase.